lib/routes/account.js

```diff
@@ -17,6 +17,7 @@ var openid = require('openid')
 var userAgent = require('../userAgent')
 var url = require('url')
 var requestHelper = require('../routes/utils/request_helper')
+var geodb = require('fxa-geodb')()
```

Comment:
Yeah, I don't know what I'm talking about. Just some Friday night Git quarter backing.
Looks like we need to merge your PR in that repo and then publish to npm before we can merge this anyways.